projects/libraw/Dockerfile

```diff
@@ -19,4 +19,7 @@ MAINTAINER jesteele@google.com
 RUN apt-get update && apt-get install -y make autoconf automake libtool pkg-config
 RUN git clone --depth 1 https://github.com/libraw/libraw
 WORKDIR libraw
-COPY build.sh libraw_fuzzer.cc $SRC/
+COPY corpus/* build.sh libraw_fuzzer.cc $SRC/
+RUN zip $SRC/libraw_cr2_fuzzer_seed_corpus.zip $SRC/canon_eos_20d.cr2
```

Comment:
Ideally yes, however the maintainers didn't seem interested in us up streaming the fuzzer changes (https://github.com/LibRaw/LibRaw/issues/295). Is there a better place these files can live?

I think this is a great use case for "cross-pollination", but am also aware its not implemented with OSS-Fuzz based on the documentation.

Reply:
These 3 combined is greater than 20 mb, these are too big files to keep it next to source. If these were few kbs, then it would have been fine. are there any test files in libraw repo, how is it running the tests without sample files ?

Reply:
I agree, I feel for the same reason the changes would not be accepted upstreamed. And unfortunately LibRaw doesn't have any full end to end tests that would require sample raw images.

 I see some other libraries download a zip of raw files during this script, is that something I can do for this? And if so, is there a recommended spot for these to live?